src/main/resources/log4j.properties

```diff
@@ -16,7 +16,7 @@
 #
 
 # Set everything to be logged to the console
-log4j.rootLogger=INFO,console
+log4j.rootLogger=INFO,console,ROLLBAR
```

Comment:
Does this work if rollbar is not in this file?

Reply:
yes